getting-started/services.md

```diff
@@ -24,9 +32,7 @@ Timescale Cloud offers the following PostgreSQL optimized database services:
 
 <ServiceOverview />
 
-This section shows you how to create a service, connect to it, create a standard PostgreSQL table, then 
-convert it into a [Hypertable][hypertables]. Anything you can do with regular PostgreSQL tables, you can 
-do with hypertables, just with better performance and improved an user experience for time-series data.
+
```

Comment:
And add an intro sentence infront of every procedure to give a longer explanation of what it is all about.